src/components/Player/Player.jsx

```diff
@@ -59,16 +64,20 @@ const Player = ({ playerId, info, params }) => {
       <div className={styles.header}>
         <PlayerHeader playerId={playerId} />
         <div style={{ marginTop: 25 }}>
-          <TabBar tabs={playerPagesMapped(params.account_id)} />
+          <TabBar tabs={playerPagesMapped(playerId)} />
         </div>
       </div>
-      {getPlayerSubroute(info, playerId)}
+      {getPlayerSubroute(info, playerId, histogramName)}
     </div>
   );
 };
 // need to fix this
 
-const mapStateToProps = (state, { params }) => ({ playerId: params.account_id, info: params.info });
+const mapStateToProps = (state, { params }) => ({
+  playerId: params.account_id,
+  info: params.info,
+  histogramName: params.histogramName,
```

Comment:
trends is going to use a similar route pattern.  You might want to genericize the name.